tools/mo/openvino/tools/mo/middle/passes/infer.py

```diff
@@ -353,3 +353,8 @@ def reverse_infer(graph: Graph, nodes: list):
                 log.debug('Inputs:')
                 log_debug_dict(node.in_nodes(), 'inputs')
 
+    for node in graph.get_op_nodes(op='Parameter'):
+        name = node.soft_get('name', node.id)
+        if not node.has_valid('shape'):
+            raise Error("Reverse infer couldn't calculate partial shape of Parameter node '{}'. "
```

Comment:
Please correct two problems:

1) We can have several Parameter nodes with undefined shapes so we should print the whole list of Parameter nodes
2) Users do not know about reverse infer and what it is. It is better to say that:
`Model Optimizer is unable to deduce input shapes for the following Parameter nodes: {}...`

Reply:
You are right, i'll fix that

Reply:
fixed, please review